src/prop/model.py

```diff
@@ -133,8 +133,12 @@ def PropagationLoss(self, f, lat1, lng1, h1, lat2, lng2, h2, land_cat=''):
         code = self.nlcdIndx.NlcdCode(lat1, lng1)
       land_cat = land_use.NlcdLandCategory(code)
     print 'Using land_cat =', land_cat
-      
-    if land_cat == 'RURAL' or h1 >= 200 or h2 >= 200:
+
+#   Calculate effective heights of tx and rx:
+    profile = self.nedIndx.Profile(lat1, lng1, lat2, lng2)
+    h1eff, h2eff = EffectiveHeights(h1, h2, profile)
+    
+    if land_cat == 'RURAL' or h1eff >= 200 or h2eff >= 200:
```

Comment:
It is true there's a lot of profile recalc. The most painful is the 80km thing, but the code doesn't have all the profile-passing you might want, either, so it ends up getting recalculated.

We could do a couple things about this. One is instead of 80km we could use profile sub-sets to the nearest 80km, which would save some recalc.

Is there a spot lower where you can eliminate a profile recalc by pulling it out here?